core/client/src/test/java/alluxio/client/file/policy/DeterministicHashPolicyTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.policy;
+
+import alluxio.Constants;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.file.policy.options.BlockLocationPolicyCreateOptions;
+import alluxio.client.file.policy.options.BlockLocationPolicyGetWorkerOptions;
+import alluxio.wire.WorkerNetAddress;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Tests {@link DeterministicHashPolicy}.
+ */
+public final class DeterministicHashPolicyTest {
+  private static final int PORT = 1;
+
+  private List<BlockWorkerInfo> mWorkerInfos;
```

Comment:
this can be `final` if you declare 
```java
private final List<BlockWorkerInfo> mWorkerInfos = new ArrayList<>();
```

even more, up to you, you can may be pull in the initial 4 `BlockWorkerInfo` instances in initialization.